QUICKSTART.md

```diff
@@ -0,0 +1,351 @@
+which should yield something like:
+
+```console
+Client Version: v1.25.3
+Kustomize Version: v4.5.7
+Server Version: v1.24.3+kcp-v0.11.0
+```
+
+Additionally, one can check the available virtual workspaces using the command:
+
+```bash
+kubectl ws tree
+```
+
+which should yield something like:
+
+```console
+.
+└── root
+    └── compute
+```
+
+## 2. Install and run **KCP-Edge**
+
+Download the kcp **KCP-Edge** binaries and scripts into a `kcp-edge` subfolder in your current working directory using the following command:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/install-kcp-edge.sh) --version v0.1.0 --folder $(pwd)/kcp-edge --create-folder
+export PATH="$PATH:$(pwd)/kcp-edge/bin"
+```
+
+Start **KCP-Edge** with the following command:
+
+```shell
+kcp-edge start --user kit
+```
+
+Check that `KCP-Edge` controllers are running with the following command:
+
+```shell
+ps aux | grep -e mailbox-controller -e placement-translator -e scheduler
+```
+
+which should yield something like:
+
+```console
+user     1892  0.0  0.3 747644 29628 pts/1    Sl   10:51   0:00 mailbox-controller --inventory-context=root --mbws-context=base -v=2
+user     1902  0.3  0.3 743652 27504 pts/1    Sl   10:51   0:02 scheduler -v 2 --root-user kcp-admin --root-cluster root --sysadm-context system:admin --sysadm-user shard-admin
+user     1912  0.3  0.5 760428 41660 pts/1    Sl   10:51   0:02 placement-translator --allclusters-context system:admin -v=2
+```
+
+Check that the Edge Service Provider Workspace (`espw`) is created with the following command:
+
+```shell
+kubectl ws tree
+```
+
+which should yield:
+
+```console
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## 3. Create a kcp-edge Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)
+
+**IMW** are used by **KCP-Edge** to store *sync targets* and *placement* objects.
+Create an **IMW** named `imw-1` with the following command:
+
+```shell
+kubectl ws root
+kubectl ws create "imw-1"
+```
+
+**WMW** are used by **KCP-Edge** to store *workloads* amd *edge placement* objects.
+Create an **WMW** named `wmw-1` in a `my-org` workspace with the following command:
+
+```shell
+kubectl ws root
+kubectl ws create "my-org"
+kcp-edge --create_wmw wmw-1  # replaces: ensure-wmw.sh "wmw-1"
+```
+
+
+## 4. Example deployment of nginx workload into a kind local cluster
+
+ 
+### a. Stand up a local florin kind cluster
+
+To install kind in your environment follow the instructions [here](https://kind.sigs.k8s.io/). To use kind, you will also need to [install docker](https://docs.docker.com/get-docker/).
+
+```shell
+kind create cluster --name florin
+```  
+
+### b. Create a sync target, placement and edge syncer for onboarding the created florin edge cluster
+
+Create a syncTarget and location inventory objects to represent the edge cluster (`florin`):
+
+```shell
+kcp-edge --create_inv_item florin  env=prod    # replaces ensure-location.sh florin  env=prod
```

Comment:
@francostellari has suggested that this makes more sense as a kubctl plugin.  I agree.  I suggest we use the following two command lines instead of the current script.

```
kubectl kcp-edge ensure-location name key=value...
kubectl kcp-edge remove-location name
```

Reply:
In my discussion with @francostellari we came up with the following draft flag names:

```
1. kcp-edge --create_wmw wmw-1
2. kcp-edge --syncer florin
3. kcp-edge --create_inv_item
```

Reply:
Earlier today we agreed on the following revised convenience command for doing this.  The general syntax is

```shell
kubectl kcp-edge prep-for-cluster (--imw ws_path | --espw ws_path | --syncer-image image_ref | -o filename)* $name key=value...
```

and in this case it could be

```shell
kubectl kcp-edge prep-for-cluster --imw root:imw-1 florin env=prod
```

Reply:
This combined command would need to be introduced by corresponding text.  Perhaps something along the lines of the following.

> The following command will create the SyncTarget and Location objects in the inventory management workspace `root:imw-1` that describe the florin cluster, prepare the central componentry to work with the syncer, and write a file containing the YAML to submit to the florin cluster to deploy the syncer there.